client/landing/stepper/declarative-flow/internals/components/help-center/async.tsx

```diff
@@ -12,8 +12,18 @@ const AsyncHelpCenter = () => {
 		setShowHelpCenter( false );
 	}, [ setShowHelpCenter ] );
 
+	/**
+	 * The ?v2 query parameter ensures Webpack treats this Help Center as separate from the one in the main client app.
+	 * Without it, Webpack would create one shared chunk, loaded in both apps. Since Stepper is smaller, more CSS would
+	 * need be bundled into that shared chunk. This is great for Stepper, but it duplicates the CSS in the main client app.
+	 * See: #97480
+	 */
 	return (
-		<AsyncLoad require="@automattic/help-center" placeholder={ null } handleClose={ handleClose } />
+		<AsyncLoad
+			require="@automattic/help-center?v2"
```

Comment:
A better name for the query param would be `?stepper`. Then the resulting chunk name is:
```
async-load-automattic-help-center-stepper
```
instead of
```
async-load-automattic-help-center-v-2
```
The chunk name is the only thing that is affected by the query param.